crux-llvm/test-data/golden/stdio/write-stdout.good

```diff
@@ -0,0 +1 @@
+[Crux] Overall status: Valid.
```

Comment:
Do we not expect to see the actual output here?

Reply:
ha ha I was wondering about that too. It turns out that the test suite only captures output from the handles passed to `Crux.mkOutputConfig`. And those both go to a special other handle in the test suite.

So we could hook the output up to those handles instead of actual stdout/stderr. I wasn't sure if that was a good idea or not, given that there seems to be other processing on those handles. I'm happy to switch it over if we think that is reasonable.

Reply:
Yeah, we should use the `mkOutputConfig` handles.  That way we can capture `stdout` into IDE windows, etc, as we go forward.

Reply:
Okay, fixed. I think there is only one handle there, unfortunately, which means stdout and stderr are combined for now.

Reply:
I think that's fine.  If we enhance the output configuration to have separate handles, we can change this then.